smarts/env/hiway_env.py

```diff
@@ -237,7 +282,11 @@ def step(
         observations, rewards, dones, extras = self._smarts.step(agent_actions)
 
         infos = {
-            agent_id: {"score": value, "env_obs": observations[agent_id]}
+            agent_id: {
+                "score": value,
+                "env_obs": observations[agent_id],
+                "done": dones[agent_id],
+            }
             for agent_id, value in extras["scores"].items()
         }
```

Comment:
Why are observation and done duplicated, and returned again in `infos`? Can they be removed from `infos`?

Reply:
Right, done for a `gym` environment should relate to the `gym` environment itself rather than the agents in it. If you notice a bit further down I now only return a `bool` rather than the original dictionary.

I remember that the observations here are duplicated as the raw observations in the case that there are wrappers applied over the environment. In my memory it seems to have been for logging purposes.